app/grants/views.py

```diff
@@ -2698,7 +2698,9 @@ def contribute_to_grants_v1(request):
             subscription.save()
 
             # step 3: create contribution + fire celery
-            contribution = subscription.create_contribution(tx_id, False)
+            contribution = subscription.create_contribution(tx_id, True)
+            contribution.success = True
```

Comment:
is it marked as failed if the contribution fails elsewhere?

Reply:
nope! there is no way for us to fail -> hence it'll be in forever pending 
The fallback we could have is to auto fail it after 5 hours